canister/src/api/get_utxos.rs

```diff
@@ -930,11 +958,14 @@ mod test {
             }
         );
 
-        // min confirmations is too large. Should panic.
-        get_utxos(GetUtxosRequest {
-            address: address_1.to_string(),
-            filter: Some(UtxosFilter::MinConfirmations(3)),
-        });
+        // min confirmations is too large. Should return error.
```

Comment:
```suggestion
        // min confirmations is too large. Should return an error.
```

Reply:
It looks like I will finally need to learn articles:)